Testing/RunUnitTests.ps1

```diff
@@ -156,7 +156,7 @@ function Invoke-ControlGroupItem {
 
             elseif(Test-Path -Path $Filename.Fullname -PathType Leaf) {
                 Write-Output "`nTesting Control Group $ControlGroup"
-                ..\opa_windows_amd64.exe test $RegoPolicyPath .\$($Filename.Fullname) $Flag
+                & $OPAExe test $RegoPolicyPath .\$($Filename.Fullname) $Flag
```

Comment:
This conflicts with the AAD PR. Either add the entire fix to this PR or remove to prevent merge conflicts.

Reply:
This PR is going to be merged first and the code change is the same as the other PR anyway.